mkdocs.yml

```diff
@@ -163,72 +163,11 @@ markdown_extensions:
 # Plugins
 plugins:
   - search
+  - awesome-pages
 
 # Page Tree/Sidebar
 # https://www.mkdocs.org/user-guide/writing-your-docs/#configure-pages-and-navigation
+# https://3os.org/utilities/markdown-cheatsheet/awesome-pages/
+# We now use awesome-pages plugin, so check `.pages` files inside renovate docs folder
```

Comment:
```suggestion
# We now use the awesome-pages plugin, which uses `.pages` files inside the renovate docs/ folder to set the navigation structure.
# The source `.pages` file(s) are in the renovatebot/renovate repository.
```

Let's explain _where_ to find the source `.pages` files, so I don't forget later. 😉 🙈